plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/connection/DBPDataSourceProviderDescriptor.java

```diff
@@ -52,6 +52,8 @@ public interface DBPDataSourceProviderDescriptor extends DBPNamedObject {
 
     DBPDataSourceProviderDescriptor getParentProvider();
 
+    List<DBPDataSourceProviderDescriptor> getChildrenProviders();
```

Comment:
Null and empty are different things, I think :(

Reply:
They are. Yet returning `null` instead of an empty list is really weird. Anyways, the only implementation returns a non-null list, so let's mark it as `@NotNull`.